ogr/ogrsf_frmts/miramon/ogrmiramonlayer.cpp

```diff
@@ -334,6 +334,8 @@ OGRMiraMonLayer::OGRMiraMonLayer(GDALDataset *poDS, const char *pszFilename,
                     delete m_poSRS;
                     m_poSRS = nullptr;
                 }
+                m_poSRS->SetAxisMappingStrategy(OAMS_TRADITIONAL_GIS_ORDER);
```

Comment:
That doesn't seem correct, as m_poSRS can have been set to NULL just above at line 337. Those 2 lines (337 and 338) should likely be put in a ``else {}`` block

Reply:
Yes, I see it later, sorry